daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/processors/VariableMap1.scala

```diff
@@ -278,8 +280,8 @@ class VariableMap private (vrds: Seq[VariableRuntimeData], vTable: Array[Seq[Var
   }
 
   def cloneForSuspension(): VariableMap = {
-    val newTable = new Array[Seq[VariableInstance]](vTable.size)
-    Array.copy(vTable, 0, newTable, 0, vTable.size)
+    val newTable = new Array[Seq[VariableInstance]](vTable.length)
+    Array.copy(vTable, 0, newTable, 0, vTable.length)
     new VariableMap(vrds, newTable)
```

Comment:
Is here a different between size vs length?

Reply:
It seems length is preferred to size in Scala code style.